datasets/us/fbi_most_wanted/crawler.py

```diff
@@ -99,10 +99,11 @@ def crawl_person(context: Context, url: str) -> None:
 
     aliases = doc.findtext('.//div[@class="wanted-person-aliases"]/p')
     if aliases is not None:
-        for alias in aliases.split(","):
-            alias = alias.strip()
-            prop = "alias" if " " in alias else "weakAlias"
-            person.add(prop, alias)
+        for raw_alias in aliases.split(","):
```

Comment:
what do you think of replacing `,”, “` with `”, “`?
![image](https://github.com/user-attachments/assets/35138ccd-2b9b-472a-affa-d9fa1a858e23)